### PR TITLE
Chika

### PR DESCRIPTION
**Pull request by ChyngyzTynybekov**

### CODE REVIEW

File: src/man.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+public class man.java{
+public static void main(String[]args){
```
Comment (kykaparov) [inline src/man.java]: 4 spaces

File: src/man.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (kykaparov) [inline src/man.java]: remove empty line

File: src/man.java (hunk 1 above)
Comment (Tarmaljigit) [reply src/man.java]: there shouldn't be an empty spaces

Comment (Gulnura2017) [reply src/man.java]: on line 5 put ';' at the end of the line

File: src/man.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    int a=5;
+    int b=2
```
Comment (Mekabek) [inline src/man.java]: After 2 put ";"

**Comment by JapanNS** [comment]
After all the commits, it looks good to me.

File: src/man.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+        int sum=a+b;
+    System.out.println(sum);
+        }
+        }
```
Comment (Mekabek) [inline src/man.java]: Chika you did it!!!

Comment (Guli555) [reply src/man.java]: Happy New Year Chika

File: src/man.java (hunk 3 above)
Comment (tyncha) [inline src/man.java]: after 2 out ;

**Pull request merged**